src/core/include/openvino/op/lstm_cell.hpp

```diff
@@ -23,6 +23,12 @@ enum class LSTMWeightsFormat {
     IOFC,  // ONNX
 };
 
+enum class LSTMPeepholesFormat {
```

Comment:
Why do we need new enums in public headers?

Reply:
@ilyachur The new enum is required to support LSTMSequence-1 in ONNX frontend. If we will not have any format indicator for Peepholes and assume only FIO format, then I can remove it.

Reply:
@itikhono Please take a look.

But any way you have the same enum in RNNCellBase, so I propose to avoid duplication and keep only one enumeration.

Reply:
@ilyachur Ok, let me check it again!

Reply:
@ilyachur I fixed it to use only single enum definition in ov::op::util name space. Thanks!